### ISSUE TITLE
gatsby-plugin-sharp: `"stripMetadata": false` option does not work with webp

### ISSUE THREAD

**Issue by kije** [gatsbyjs/gatsby#20709]
## Description

When setting `stripMetadata` to false in the `gatsby-plugin-sharp` options in `gatsby-config.js`, metadata and embedded ICC profiles are nevertheless stripped from the generated WebP files.

I took a look at the corresponding code, the problem seems to be that the `metadata` option is not provided to imagemin-webp (which defaults to `metadata: 'none'`, resulting in the metadata getting removed from the image -> see https://www.npmjs.com/package/imagemin-webp?activeTab=readme)

### Steps to reproduce

1. Setup Gatsby
2. Install `gatsby-plugin-sharp` 
3. Set `stripMetadata` to false on the `gatsby-plugin-sharp` options in `gatsby-config.js`
4. Create a page or component using `gatsby-plugin-sharp` to transform an image to WebP from a source file containing metadata/an embedded ICC profile
5. Check generated WebP file for metadata/embedded ICC profile

### Expected result

When setting `stripMetadata` to false, generated WebP files should also retain metadata/embedded ICC profiles.

### Actual result

Existing metadata/ICC profiles are stripped from generated WebP files

### Environment
```
  System:
    OS: Linux 5.3 Pop!_OS 19.10
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
    Shell: 5.7.1 - /usr/bin/zsh
  Binaries:
    Node: 13.0.1 - ~/.nvm/versions/node/v13.0.1/bin/node
    Yarn: 1.19.2 - /usr/bin/yarn
    npm: 6.12.0 - ~/.nvm/versions/node/v13.0.1/bin/npm
  Languages:
    Python: 2.7.17 - /usr/bin/python
  Browsers:
    Chrome: 78.0.3904.108
    Firefox: 70.0.1
  npmPackages:
    gatsby: ^2.18.25 => 2.18.25
    gatsby-background-image: ^0.9.12 => 0.9.12
    gatsby-env-variables: ^1.0.4 => 1.0.4
    gatsby-image: ^2.2.39 => 2.2.39
    gatsby-plugin-google-analytics: ^2.1.33 => 2.1.33
    gatsby-plugin-lodash: ^3.1.19 => 3.1.19
    gatsby-plugin-manifest: ^2.2.37 => 2.2.37
    gatsby-plugin-netlify: ^2.1.31 => 2.1.31
    gatsby-plugin-netlify-cache: ^1.2.0 => 1.2.0
    gatsby-plugin-nprogress: ^2.1.18 => 2.1.18
    gatsby-plugin-offline: ^3.0.32 => 3.0.32
    gatsby-plugin-prefetch-google-fonts: ^1.4.3 => 1.4.3
    gatsby-plugin-purgecss: ^4.0.1 => 4.0.1
    gatsby-plugin-react-helmet: ^3.1.21 => 3.1.21
    gatsby-plugin-react-helmet-canonical-urls: ^1.4.0 => 1.4.0
    gatsby-plugin-remove-trailing-slashes: ^2.1.20 => 2.1.20
    gatsby-plugin-robots-txt: ^1.5.0 => 1.5.0
    gatsby-plugin-sass: ^2.1.27 => 2.1.27
    gatsby-plugin-sharp: ^2.3.13 => 2.3.13
    gatsby-plugin-sitemap: ^2.2.26 => 2.2.26
    gatsby-plugin-transition-link: ^1.17.7 => 1.17.7
    gatsby-plugin-ts: ^2.1.1 => 2.1.1
    gatsby-source-filesystem: ^2.1.46 => 2.1.46
    gatsby-source-graphql: ^2.1.31 => 2.1.31
    gatsby-transformer-sharp: ^2.3.13 => 2.3.13
  npmGlobalPackages:
    gatsby-cli: 2.8.22
```

**Comment by LekoArts** [comment]
Thanks for the issue! We'd be happy to receive a PR fixing this 👍

**Comment by rajatsharma** [comment]
This PR #20860 fixes this

**Comment by pieh** [comment]
Published `gatsby-plugin-sharp@2.4.2`
